### DISCUSSION TITLE
MacOS X 10.11 email client - SSL: error:0A000102:SSL routines::unsupported protocol

### DISCUSSION THREAD

**Discussion by pini-gh** [Mailu/Mailu#3067] [User support]
## Environment & Version

- `docker compose version: 1.25.0`
- Version: `2.0.34`

## Description
A user with a Macbook running MacOS X 10.11 and MacOS email client "Mail" version 9.3 cannot connect to the mail server.

## Replication Steps
Configure an email account on the above described system.

## Observed behaviour
The email client complains il cannot connect to the server to fetch the emails.

## Expected behaviour
Connexction and email synchronization should work

## Logs
`mailu_front_1` logs:
```
2023/11/19 17:21:16 [info] 1327#1327: *1656 SSL_do_handshake() failed (SSL: error:0A000102:SSL routines::unsupported protocol) while SSL handshaking, client: <redacted>, server: 0.0.0.0:993
```

## What I've tried so far
I've tried setting `ssl_protocols TLSv1 TLSv1.1 TLSv1.2 TLSv1.3;` in `/etc/nginx/tls.conf` in the `mailu_front_1` container. With no chance.

Unfortunately, setting log level to DEBUG doesn't give more details.

Any hint appreciated.

**Comment by nextgens** [comment]
You will need to tweak ssl_ciphers too for it to work. See the OLD profile on https://ssl-config.mozilla.org/
